src/commands/reminder/CreateReminder.ts

```diff
@@ -0,0 +1,170 @@
+import BaseCommand, { ArgumentType, ICommandContext, } from "../BaseCommand";
+
+import { AdvancedCollector, GeneralUtilities, TimeUtilities, } from "../../utilities";
+import { ButtonBuilder, ButtonStyle, ColorResolvable, TextInputBuilder, TextInputStyle } from "discord.js";
+import { PostGresReminder } from "../../utilities/PostGresReminder";
+
+export default class CreateReminder extends BaseCommand {
+    public constructor() {
+        super ({
+            cmdCode: "CREATEREMINDER",
+            formalCommandName: "CreateReminder",
+            botCommandName: "createreminder",
+            description: "Create a reminder.",
+            generalPermissions: [],
+            botPermissions: [],
+            argumentInfo: [
+                {
+                    displayName: "Reminder Information", // what information the user would want to be reminded about
+                    argName: "reminder_info",
+                    type: ArgumentType.String,
+                    desc: "The information you'd like to be reminded about.",
+                    required: true,
+                    example: ["CSE 100 homework due"],
+                },
+            ],
+            commandCooldown: 4 * 1000,
+            guildOnly: false,
+            botOwnerOnly: false,
+        });
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public async run(ctx: ICommandContext): Promise<number> {
+        const reminderMsg = ctx.interaction.options.getString("reminder_info"); // get user input
+        // unique ID for the message (per person)
+        const uniqueId = `${Date.now()}_${ctx.user.id}_${Math.random()}`;
+
+        if (reminderMsg) {
```

Comment:
You don't need to check if `reminderMsg` is `null`/`undefined` since, by the argument list, `reminderMsg` is _required_.